workflow-runtime/src/commonTest/kotlin/com/squareup/workflow1/internal/SubtreeManagerTest.kt

```diff
@@ -167,9 +168,9 @@ internal class SubtreeManagerTest {
       assertFalse(tickOutput.isCompleted)
 
       eventHandler("event!")
-      val update = tickOutput.await()!!.value
+      val update = tickOutput.await().value
 
-      val (_, output) = update.applyTo("props", "state")
+      val (_, output) = update!!.applyTo("props", "state")
```

Comment:
Nit: seems like it'd be nicer to put the `!!` in the declaration, like it was originally. Same for the two `initialAction!!` bits below.